components/brave_wallet/browser/brave_wallet_prefs.cc

```diff
@@ -58,6 +58,17 @@ base::Value GetDefaultUserAssets() {
   return user_assets_pref;
 }
 
+base::Value GetDefaultSelectedNetworks() {
+  base::Value selected_networks(base::Value::Type::DICTIONARY);
+  selected_networks.SetStringKey(brave_wallet::kEthereumPrefKey,
+                                 brave_wallet::mojom::kMainnetChainId);
+  selected_networks.SetStringKey(brave_wallet::kSolanaPrefKey,
+                                 brave_wallet::mojom::kSolanaMainnet);
+  // TODO(spylogsster): Add Filecoin
```

Comment:
Just double checking, we didn't add brave_wallet::mojom::kFilecoinMainnet here because we don't have network kKnownFilNetworks yet?

Reply:
Yep, that is right